packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -131,7 +131,7 @@ export default class Node extends PureComponent {
                     isHidden={$get('properties._hidden', node)}
                     isHiddenInIndex={$get('properties._hiddenInIndex', node)}
                     hasError={this.hasError()}
-                    label={$get('label', node)}
+                    label={$get('label', node).replace(/<\/?[^>]+(>|$)/g, '')}
```

Comment:
Then I would use a npm lib for this, am I right that I have to put this dependency at the  rootlevel AND the package which uses it?